src/woo_search/search_index/tasks.py

```diff
@@ -23,6 +23,27 @@
 
 logger = logging.getLogger(__name__)
 
+# TODO: replace this with some code which keeps in sync with gpp-app
+SUPPORTED_FILE_MIMETYPES = [
+    "text/csv",
+    "application/vnd.ms-excel",
+    "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+    "text/html",
+    "application/vnd.oasis.opendocument.formula",
+    "application/vnd.oasis.opendocument.presentation",
+    "application/vnd.oasis.opendocument.spreadsheet",
+    "application/vnd.oasis.opendocument.text",
+    "application/pdf",
+    "text/plain",
+    "application/vnd.openxmlformats-officedocument.presentationml.slideshow",
+    "application/vnd.ms-powerpoint",
+    "application/vnd.openxmlformats-officedocument.presentationml.presentation",
+    "application/vnd.ms-powerpoint.slideshow.macroEnabled.12",
+    "application/rtf",
+    "application/msword",
+    "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+]
```

Comment:
We need to do something better with this in the future but for now we can just hardcode it imo.

Reply:
Created #90 for this - will backport this solution and then for the main branch use the proper setup!